src/main/java/core/basesyntax/service/file/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.file;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void write(String data, String path) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(path))) {
+            writer.write(data);
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot write to file: " + path, e);
```

Comment:
Consider whether wrapping the `IOException` in a `RuntimeException` is the best approach for your application. Depending on your error handling strategy, you might want to define a custom exception or handle the error in a way that allows for recovery or logging.